addons/account/models/account_tax.py

```diff
@@ -105,6 +105,7 @@ def _default_tax_group(self):
     sequence = fields.Integer(required=True, default=1,
         help="The sequence field is used to define order in which the tax lines are applied.")
     amount = fields.Float(required=True, digits=(16, 4), default=0.0)
+    real_amount = fields.Float(string='Real amount to apply', compute='_compute_real_amount', store=True)
```

Comment:
shouldn't this be in point_of_sale module?

Reply:
pim just told me to put it in account module